### PR TITLE
Store property tables separately in subtree JSON

### PR DESCRIPTION
**Pull request by j9liu**
This PR implements the changes in the 3D Tiles repository merged in [#642](https://github.com/CesiumGS/3d-tiles/pull/642). Property tables are now stored under their own property in the subtree JSON. `tileMetadata` and `contentMetadata` are now an index and array of indices respectively.

cc @ptrgags

### CODE REVIEW

File: Source/Scene/ImplicitSubtree.js [hunk 1]
```diff
@@ -337,14 +337,23 @@ function initialize(subtree, json, subtreeView, implicitTileset) {
...
+    const length = subtreeJson.contentMetadata.length;
+    for (let i = 0; i < length; i++) {
+      const propertyTableIndex = subtreeJson.contentMetadata[i];
+      contentPropertyTableJsons.push(
+        subtreeJson.propertyTables[propertyTableIndex]
+      );
+    }
+  }
+
+  // TODO: FIGURE OUT FAILURE IN IMPLICIT SUBTREE SPEC
```
Comment (ptrgags) [inline Source/Scene/ImplicitSubtree.js:356]: Please remove this TODO (assuming this was addressed?)

**Review by ptrgags — COMMENTED** [review]
@j9liu code looks good, one comment.

File: Source/Scene/ImplicitSubtree.js (hunk 1 above)
Comment (j9liu) [reply Source/Scene/ImplicitSubtree.js:356]: whoops! thanks for the catch

**Comment by ptrgags** [comment]
@j9liu thanks, merged!

**Pull request merged**